lib/pkgcloud/common/auth.js

```diff
@@ -24,7 +24,9 @@ auth.basic = function basicAuth(req) {
 // Add Account number for requests to rackspace API
 auth.accountId = function (req) {
   req.headers = req.headers || {};
-  req.headers['x-auth-project-id'] = this.config.accountNumber;
+  if (this.config.accountNumber) {
+    req.headers['x-auth-project-id'] = this.config.accountNumber;
+  }
```

Comment:
I have a feeling this code path isn't even used anywhere anymore. Although I can't recall for certain.